docs/resources/mws_networks.md

```diff
@@ -26,9 +26,9 @@ Please follow this [complete runnable example](../guides/aws-workspace.md) with
 
 Use this resource to [configure VPC](https://docs.gcp.databricks.com/administration-guide/cloud-configurations/gcp/customer-managed-vpc.html) & subnet for new workspaces within GCP. It is essential to understand that this will require you to configure your provider separately for the multiple workspaces resources.
 
-* Databricks must have access to a subnet in the same region as the workspace, of which IP range will be used to allocate your workspace’s GKE cluster nodes.
+* Databricks must have access to a subnet in the same region as the workspace, of which IP range will be used to allocate your workspace’s GCE cluster nodes.
 * The subnet must have a netmask between /29 and /9.
-* Databricks must have access to 2 secondary IP ranges, one between /21 to /9 for workspace’s GKE cluster pods, and one between /27 to /16 for workspace’s GKE cluster services.
+* Databricks must have access to 2 secondary IP ranges, one between /21 to /9 for cluster pods, and one between /27 to /16 for cluster services.
```

Comment:
We need only one subnet, no secondary IP ranges

Reply:
Ah OK, thanks, will remove this.